trade-finance-manager-api/src/v1/rest-mappings/mappings/facilities/mapFacilityType.js

```diff
@@ -1,49 +0,0 @@
-const { FACILITY_TYPE, MAPPED_FACILITY_TYPE } = require('@ukef/dtfs2-common');
-const CONSTANTS = require('../../../../constants');
-
-/**
- * Maps facility type to a different, tfm-api specific facility type.
- * Note there is a difference between this facility type, the input facility type,
- * and how the facility type is displayed in the front end.
- *
- * @returns {string} the mapped facility type
- *
- * @example
- * // returns 'Bid Bond'
- * mapFacilityType({...facility, code: 'BSS', bondType: 'Bid Bond'})
- *
- * @example
- * // returns 'Loan'
- * mapFacilityType({...facility, code: 'EWCS'})
- *
- * @example
- * // returns 'Cash Facility'
- * mapFacilityType({...facility, ukefFacilityType: 'Cash'})
- *
- * @example
- * // returns 'Contingent Facility'
- * mapFacilityType({...facility, ukefFacilityType: 'Contingent'})
- */
-const mapFacilityType = (facility) => {
-  const { facilityProduct, ukefFacilityType } = facility;
-
-  if (facilityProduct.code === CONSTANTS.FACILITIES.FACILITY_PRODUCT_CODE.BOND) {
```

Comment:
Previously it was using the derived 'facilityProduct' on the `clonedFacilitySnapshot` to determine if it was a BOND or a LOAN facility. However if you look at the `mapFacilityProduct` file (also shows as one of the changed files so hopefully easy to locate), this is precisely equivalent to the original facility snapshot's `type` being 'Bond' or 'Loan'. 

So in the new TS file I've just based this logic of the `type` instead of using the `code` as it's doing the same things but easier to read and reduces dependency on the `clonedFacilitySnapshot`